src/main/python/main/ayab/menu.py

```
@@ -52,10 +51,50 @@ def __init__(self, parent: GuiMain):
+        while i < self.parent().prefs.MAX_RECENT_COUNT and i < len(self.parent().prefs.recentFiles):
+            self.recents[i].setText(self.parent().prefs.recentFiles[i])
+            self.recents[i].setVisible(True)
+            i += 1
+
+        # Set visible toplevel menu item if there are recent files available
+        if i > 0:
+            self.ui.menu_recent_files.menuAction().setVisible(True)
+
+    # Function is a recent menu click handler
+    # Load image from file
+    def load_recent(self, e) -> None:
```

Ruff (0.15.1)

[warning] Unused method argument: `e`

(ARG002)